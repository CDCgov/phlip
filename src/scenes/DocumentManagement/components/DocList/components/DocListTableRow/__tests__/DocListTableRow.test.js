import React from 'react'
import { shallow } from 'enzyme'
import { DocListTableRow } from '../index'

const props = {
  doc: {
    name: 'doc1',
    uploadedBy: { firstName: 'Test', lastName: 'User' },
    uploadedDate: '1/1/2000',
    projects: [],
    jurisdictions: []
  },
  projectList: ['P1', 'P2'],
  jurisdictionList: ['J1', 'J2'],
  onSelectFile: () => {},
  isChecked: false
}

describe('DocumentManagement - DocList - DocListTableRow', () => {
  test('should render correctly', () => {
    expect(shallow(<DocListTableRow {...props} />)).toMatchSnapshot()
  })
})