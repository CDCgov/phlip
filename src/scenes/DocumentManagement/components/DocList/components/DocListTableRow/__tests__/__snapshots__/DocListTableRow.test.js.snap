// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentManagement - DocList - DocListTableRow should render correctly 1`] = `
<WithStyles(TableRow)>
  <WithStyles(TableCell)
    style={
      Object {
        "paddingLeft": 24,
        "paddingRight": 0,
        "width": "1%",
      }
    }
  >
    <WithStyles(CheckboxLabel)
      input={
        Object {
          "onChange": [Function],
          "value": false,
        }
      }
    />
  </WithStyles(TableCell)>
  <WithStyles(TableCell)
    padding="checkbox"
    style={
      Object {
        "maxWidth": 250,
        "overflow": "hidden",
        "padding": 18,
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
        "width": 250,
      }
    }
  >
    <WithTheme(TextLink)
      to={
        Object {
          "pathname": "/docs/undefined/view",
          "state": Object {
            "document": Object {
              "jurisdictions": Array [
                "Ohio (state)",
                "Georgia (state)",
              ],
              "name": "doc1",
              "projects": Array [
                "Project 1",
                "Project 2",
              ],
              "uploadedBy": Object {
                "firstName": "Test",
                "lastName": "User",
              },
              "uploadedDate": "2000-1-1",
            },
          },
        }
      }
    >
      doc1
    </WithTheme(TextLink)>
  </WithStyles(TableCell)>
  <WithStyles(TableCell)
    padding="checkbox"
    style={
      Object {
        "width": 90,
      }
    }
  >
    Test
     
    User
  </WithStyles(TableCell)>
  <WithStyles(TableCell)
    padding="checkbox"
    style={
      Object {
        "width": 50,
      }
    }
  >
    12/31/1999
  </WithStyles(TableCell)>
  <WithStyles(TableCell)
    padding="checkbox"
    style={
      Object {
        "maxWidth": 250,
        "overflow": "hidden",
        "padding": 18,
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
        "width": 250,
      }
    }
  >
    P1, P2
  </WithStyles(TableCell)>
  <WithStyles(TableCell)
    padding="checkbox"
    style={
      Object {
        "maxWidth": 250,
        "overflow": "hidden",
        "padding": 18,
        "paddingRight": 24,
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
        "width": 250,
      }
    }
  >
    J1, J2
  </WithStyles(TableCell)>
</WithStyles(TableRow)>
`;
